host/libvirt_lxc_backend.go

```diff
@@ -669,11 +672,28 @@ func (c *libvirtContainer) watch(ready chan<- error) error {
 			g.Log(grohl.Data{"at": "get_streams", "status": "error", "err": err.Error()})
 			return err
 		}
+
 		log := c.l.openLog(c.job.ID)
 		defer log.Close()
-		// TODO: log errors from these
-		go log.Follow(1, stdout)
-		go log.Follow(2, stderr)
+
+		muxConfig := logmux.Config{
+			AppName: c.job.Metadata["flynn-controller.app"],
```

Comment:
actually, I've noticed an issue with metadata when writing integration tests. Here's another bit of metadata for a local job:

```
ENV[FLYNN_JOB_ID]          96a3d2f495414019a73f7254fbc225fc
```

We're sending logs to the log aggregator with the job ID as `96a3d2f495414019a73f7254fbc225fc`, but afaict the external representation of that is actually `flynn-96a3d2f495414019a73f7254fbc225fc`. So as of now, in order to actually view the logs for a job, I have to run:

```
flynn log -j 96a3d2f495414019a73f7254fbc225fc
```

Rather than using the identifier shown in `flynn ps`, like this:

```
flynn log -j flynn-96a3d2f495414019a73f7254fbc225fc
```

Where should that transformation take place? Should we be adding a `flynn-` prefix before the logs get sent to the aggregator, or should we be stripping it when requesting logs from the aggregator via the controller?